### ISSUE TITLE
Please build a preview release. 

### ISSUE THREAD

**Issue by KurioHonoo** [Genymobile/scrcpy#629]
Those of us on Android Q beta can't use scrcpy until a new build is released with the recent Q commits. 

I've tried building it multiple times and constantly run into issues as the instructions leave out a ton of important steps. 

Thanks for the awesome work thus far!

**Comment by rom1v** [comment]
> Those of us on Android Q beta can't use scrcpy

Why that? Only the "turn screen off" feature should be broken on Android Q in v1.9. Is there anything else?

**Comment by KurioHonoo** [comment]
> Why that? Only the "turn screen off" feature should be broken on Android Q in v1.9. Is there anything else?
So it was! I was having weirder display issues, testing it again and everything seems to be working much better. Not sure what I was thinking that I got that all mixed up. 

> For which platform do you try to build? What are the problems you encounter?
I was using the latest release of Ubuntu. After multiple attempts I actually got everything released on it and was able to build the app, but I use Windows on my main machine that I want it on so that was a waste. I have yet to be able to get the MakeCrossfile.Windows work and just gave up after a while. Ran into too many java related issues or server location issues. 

I will probably start over and try again sometime in the future after I'm less annoyed at it. My main point of contention was that the wiki specifically states "Don't worry, it's not that hard" and then it ends up being much harder because there are a lot of in between steps that aren't included so you're constantly trying to install something new or configure something else.
